### PR TITLE
Add score to cross-posts

### PR DESCRIPTION
**Pull request by micahmo**
## Pull Request Description

This PR adds scores to cross-posts. This is something I've been wanting for a while because I often come across a post with cross-posts and I want to view the one with the most engagement. The cross-posts are sorted by votes, but I can't tell if the one I'm looking at or one of the ones listed in the cross-posts area has more votes. Now that is possible to see, and it can help to inform whether I am interested in viewing a certain cross-post.

Note: I'm not sure whether any other Lemmy UI/client does this, but there is precedent from the Mbin UI.

| image |
| - |

## Issue Being Fixed

Issue Number: N/A

## Screenshots / Recordings

https://github.com/thunder-app/thunder/assets/7417301/c266edc7-8df6-4c7f-8eca-6c71cac45da4

## Checklist

- [ ] Did you update CHANGELOG.md?
- [ ] Did you use localized strings where applicable?
- [ ] Did you add `semanticLabel`s where applicable for accessibility?

### CODE REVIEW

**Review by ggichure — APPROVED** [review]

**Review by hjiangsu — APPROVED** [review]
LGTM!

I'll hold off on merging PRs for now since I wanna get a nightly build out in the next couple of days. I'll approve it so that it can be merged in right after the next nightly build!

**Comment by hjiangsu** [comment]
I might hold off on merging this PR since @CTalvio is working on some re-designs as well. Thoughts @micahmo?

**Pull request closed without merging**
